sklearn/naive_bayes.py

```diff
@@ -484,13 +484,13 @@ def partial_fit(self, X, y, classes=None, sample_weight=None):
         y : array-like, shape = [n_samples]
             Target values.
 
-        classes : array-like, shape = [n_classes], optional (default=None)
+        classes : array-like, shape = [n_classes] (default=None)
```

Comment:
well I could change that but that would really increase the size of the pull request, even if I just do it for naive_bayes.py